pygsti/forwardsims/termforwardsim_calc_statevec.pyx

```diff
@@ -1531,654 +1531,3 @@ def circuit_achieved_and_max_sopm(fwdsim, rholabel, elabels, circuit, repcache,
         max_sopm[i] = max_sum_of_pathmags[i]
 
     return achieved_sopm, max_sopm
-
-
```

Comment:
See note in pygsti/forwardsims/termforwardsim.py

Reply:
@sserita, do you mean the note which I've commented on here: https://github.com/sandialabs/pyGSTi/pull/452/files/00982b291c80a972790cf09dfaa4cf556ad1d9da#r1878653983 ?